spec/lib/stripe/credit_card_remover_spec.rb

```diff
@@ -4,16 +4,12 @@
 require 'stripe/credit_card_remover'
 
 describe Stripe::CreditCardRemover do
-  let!(:secret) { ENV.fetch('STRIPE_SECRET_TEST_API_KEY', nil) }
```

Comment:
This pattern is used a few files : 
- spec/services/stripe_payment_status_spec.rb
- spec/controllers/spree/credit_cards_controller_spec.rb
- spec/models/stripe_account_spec.rb
- spec/models/spree/gateway/stripe_sca_spec.rb
- spec/lib/stripe/payment_intent_validator_spec.rb
- spec/lib/stripe/credit_card_cloner_spec.rb

We should update them as well, or at least create a ticket to address them later.

Reply:
Perfect, thanks for that reminder. I was waiting till this PR merged to take that action separately. But yes, indeed a follow-up issue/PR.